nucliadb/nucliadb/ingest/tests/test_utils.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2021 Bosutech XXI S.L.
+#
+# nucliadb is offered under the AGPL v3.0 and as commercial software.
+# For commercial licensing, contact us at info@nuclia.com.
+#
+# AGPL:
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from nucliadb.ingest.utils import get_driver, settings
+from nucliadb_utils.store import MAIN
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Ah nice, will `autouse=True` inject this fixture in all tests?

Reply:
Yes, just in all tests for this file.